zh_tw.lang

```diff
@@ -0,0 +1,209 @@
+gui.xaero_confirm =確認
```

Comment:
Ok, there definitely isn't supposed to a space before the "=" on every line. Please fix that.